### ISSUE TITLE
Bank autocheck: unhandleled timeout

### ISSUE THREAD

**Issue by vohmar** [internetee/registry#1474]
We have this kind of errors in autocheck logs:

```
rake aborted!
Net::OpenTimeout: execution expired
/home/registry/registry/shared/bundle/ruby/2.4.0/gems/socksify-1.7.1/lib/socksify.rb:178:in `initialize'
/home/registry/registry/shared/bundle/ruby/2.4.0/gems/socksify-1.7.1/lib/socksify.rb:178:in `initialize'
/home/registry/registry/shared/bundle/ruby/2.4.0/bundler/gems/lhv-e211516bc5ff/lib/lhv/connect_api.rb:26:in `credit_debi
t_notification_messages'
/home/registry/registry/releases/110/lib/tasks/invoices/process_payments.rake:20:in `block (2 levels) in <top (required)
>'
/home/registry/registry/shared/bundle/ruby/2.4.0/gems/airbrake-9.4.5/lib/airbrake/rake.rb:17:in `execute'
/home/registry/registry/releases/110/vendor/bundle/ruby/2.4.0/gems/rake-13.0.0/exe/rake:27:in `<top (required)>'
bin/bundle:3:in `load'
bin/bundle:3:in `<main>'
Tasks: TOP => invoices:process_payments
(See full trace by running task with --trace)
```

Should be handled more elegantly

**Comment by vohmar** [comment]
fixed by https://github.com/internetee/lhv/pull/2
needs gem update on registry side

**Comment by vohmar** [comment]
waiting for gem update on registry side

**Comment by vohmar** [comment]
closed by #1514
